software/o_c_REV/APP_POLYLFO.ino

```diff
@@ -124,7 +129,8 @@ SETTINGS_DECLARE(PolyLfo, POLYLFO_SETTING_LAST) {
   { 0, 0, 1, "B XOR A", OC::Strings::no_yes, settings::STORAGE_TYPE_U4 },
   { 0, 0, 1, "C XOR B", OC::Strings::no_yes, settings::STORAGE_TYPE_U4 },
   { 0, 0, 1, "D XOR C", OC::Strings::no_yes, settings::STORAGE_TYPE_U4 }, 
-};
+  { 8, 7, 18, "XOR bit depth", NULL, settings::STORAGE_TYPE_U4 }, 
```

Comment:
Crap! 18 is supposed to be 16. 16 won't fit either in U4, will it. Better make it U8. Or set range 6 to 15 and add 1 in frames_polylfo.cpp (the values are arbitrary from the end user POV).

Is there an easy way to see how much EEPROM storage is left? Ah, silly, that information is sent to the serial port, isn't it!

----
t

> On 3 Apr 2016, at 16:13, Patrick Dowling <notifications@github.com> wrote:
> 
> In software/o_c_REV/APP_POLYLFO.ino:
> 
> > @@ -124,7 +129,8 @@ SETTINGS_DECLARE(PolyLfo, POLYLFO_SETTING_LAST) {
> >    { 0, 0, 1, "B XOR A", OC::Strings::no_yes, settings::STORAGE_TYPE_U4 },
> >    { 0, 0, 1, "C XOR B", OC::Strings::no_yes, settings::STORAGE_TYPE_U4 },
> >    { 0, 0, 1, "D XOR C", OC::Strings::no_yes, settings::STORAGE_TYPE_U4 }, 
> > -};
> > +  { 8, 7, 18, "XOR bit depth", NULL, settings::STORAGE_TYPE_U4 }, 
> 18 > range of U4 [0;16)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>